src/Console/BaseCommand.php

```diff
@@ -45,7 +45,7 @@ abstract class BaseCommand implements CommandInterface
     public function setName(string $name)
     {
         assert(
-            strpos($name, ' '),
+            strpos($name, ' ') !== false,
```

Comment:
Existing assert was already correct. A space at 1st position would also be an invalid name.